src/alda/parser.clj

```diff
@@ -20,16 +20,12 @@
           :slur              (constantly :slur)
           :flat              (constantly :flat)
           :sharp             (constantly :sharp)
+          :natural           (constantly :natural)
           :dots              #(hash-map :dots (count %))
           :note-length       #(list* 'alda.lisp/note-length %&)
           :duration          #(list* 'alda.lisp/duration %&)
-          :pitch             (fn [s]
-                               (list* 'alda.lisp/pitch
-                                      (keyword (str (first s)))
-                                      (map #(case %
-                                              \- :flat
-                                              \+ :sharp)
-                                           (rest s))))
+          :pitch             #(list* 'alda.lisp/pitch %&)
+          :pitch-notation    #(list* 'alda.lisp/parse-pitch %)
```

Comment:
Ohhh, I see. Because `<pitch-notation>` is now a hidden node, when I write `(key c++ a-)`, the parse-pitch function will get `('c' '+' '+' 'a' '-')`, whereas I was thinking it would get something more like `(('c' '+' '+') ('a' '-'))`.  I was confused about how each separate pitch notation got to `parse-key-signature`, but I think I get it now. Thanks!